extensions/sftp/processors/SFTPProcessorBase.cpp

```diff
@@ -48,59 +48,74 @@ namespace nifi {
+    ->withDescription("Username")
+    ->isRequired(true)->supportsExpressionLanguage(true)->build());
+
+core::Property SFTPProcessorBase::Password(core::PropertyBuilder::createProperty("Password")
+    ->withDescription("Password for the user account")
+    ->isRequired(false)->supportsExpressionLanguage(true)->build());
+
+core::Property SFTPProcessorBase::PrivateKeyPath(core::PropertyBuilder::createProperty("Private Key Path")
+    ->withDescription("The fully qualified path to the Private Key file")
+    ->isRequired(false)->supportsExpressionLanguage(true)->build());
+
+core::Property SFTPProcessorBase::PrivateKeyPassphrase(core::PropertyBuilder::createProperty("Private Key Passphrase")
+    ->withDescription("Password for the private key")
+    ->isRequired(false)->supportsExpressionLanguage(true)->build());
+
+core::Property SFTPProcessorBase::StrictHostKeyChecking(core::PropertyBuilder::createProperty("Strict Host Key Checking")
+    ->withDescription("Indicates whether or not strict enforcement of hosts keys should be applied")
+    ->isRequired(true)->withDefaultValue<bool>(false)->build());
+
+core::Property SFTPProcessorBase::HostKeyFile(core::PropertyBuilder::createProperty("Host Key File")
+    ->withDescription("If supplied, the given file will be used as the Host Key; otherwise, no use host key file will be used")
+    ->isRequired(false)->build());
+
+core::Property SFTPProcessorBase::ConnectionTimeout(core::PropertyBuilder::createProperty("Connection Timeout")
+    ->withDescription("Amount of time to wait before timing out while creating a connection")
+    ->isRequired(true)->withDefaultValue<core::TimePeriodValue>("30 sec")->build());
+
+core::Property SFTPProcessorBase::DataTimeout(core::PropertyBuilder::createProperty("Data Timeout")
+    ->withDescription("When transferring a file between the local and remote system, this value specifies how long is allowed to elapse without any data being transferred between systems")
+    ->isRequired(true)->withDefaultValue<core::TimePeriodValue>("30 sec")->build());
+
+core::Property SFTPProcessorBase::SendKeepaliveOnTimeout(core::PropertyBuilder::createProperty("Send Keep Alive On Timeout")
+    ->withDescription("Indicates whether or not to send a single Keep Alive message when SSH socket times out")
+    ->isRequired(true)->withDefaultValue<bool>(true)->build());
+
+core::Property SFTPProcessorBase::ProxyType(core::PropertyBuilder::createProperty("Proxy Type")
+    ->withDescription("Specifies the Proxy Configuration Controller Service to proxy network requests. If set, it supersedes proxy settings configured per component. "
+                       "Supported proxies: HTTP + AuthN, SOCKS + AuthN")
+    ->isRequired(false)
+    ->withAllowableValues<std::string>({PROXY_TYPE_DIRECT,
                                             PROXY_TYPE_HTTP,
                                             PROXY_TYPE_SOCKS})
```

Comment:
these should line up:
```suggestion
    ->withAllowableValues<std::string>({PROXY_TYPE_DIRECT,
                                        PROXY_TYPE_HTTP,
                                        PROXY_TYPE_SOCKS})
```